docs/framework/guides/contributing/package-metadata.md

```diff
@@ -0,0 +1,63 @@
+* `className` &ndash; The name of the class used to create the plugin. This class should be exported from the file using the `export default` syntax.
+* `requires` &ndash; An array of the plugin's soft requirements and other non-explicit requirements. It should contain class names of plugins that should be included if this plugin is added.
+* `registeredToolbars` &ndash; An array of all toolbar names registered by the plugin. These names need to represent the configuration path (e.g. `table.contentToolbar` for `editorConfig.table.contentToolbar` and `table.tableToolbar` for the `editorConfig.table.tableToolbar`, which are registered by the `Table` plugin).
+* `uiComponents` &ndash; An array of objects, that describes UI components exported by the plugin. Each object in this array may contain:
+	* `name` &ndash; A name of the component the plugin exports. It should match the actual UI name registered by the plugin.
+	* `type` &ndash; Component type: `Button`, `SplitButton` or `Dropdown`.
+	* `iconPath` &ndash; A path to the SVG icon for `Button` or `SplitButton`. The path can be either relative to the package or absolute - linking to a resource from another package.
+	* `label` &ndash; Text content for `Dropdown` components.
+	* `toolbars` &ndash; An array of toolbar names, a given UI component can be added to. Some UI components may be added to multiple toolbars.
+* `htmlOutput` &ndash; An array of objects, that defines all possible HTML elements, which can be created by a given plugin. The main property in this object is `elements`. Other properties (e.g. `classes`, `styles`, `attributes`) only apply to items defined in the `elements` property within a given object. Wildcard character `*` is used to mark any value. Full list of all these properties includes:
+	* `elements` &ndash; HTML elements (a single one or an array of these) that are created or altered by the plugin. The pseudo-element `$block` indicates that a given plugin applies classes, styles or attributes (defined in appropriate properties) for all block elements.
+	* `classes` &ndash; CSS class names (a single one or an array of these) that may be applied to the HTML elements defined in the `elements` property.
+	* `styles` &ndash; Inline CSS styles (a single one or an array of these) that may be applied to the HTML elements defined in the `elements` property.
+	* `attributes` &ndash; HTML attributes (a single one or an array of these) other than `class` and `styles` (covered separately), that might be applied to the HTML elements defined in the `elements` property.
+	* `implements` &ndash; A name of an element or a pseudo-element which indicates that HTML elements defined in the `elements` property may contain classes, styles or attributes that are created by other plugins. For example, `implements` equal to `$block` means, that HTML elements may contain classes, styles or attributes that are defined by another plugin, which have `elements` equal to `$block`.
```

Comment:
Here:  
  
\> classes, styles or attributes that are defined by another plugin, which have (...)  
  
if it's about the classes etc. or  
  
\> classes, styles or attributes that are defined by another plugin, which has (...)  
  
if it's about the plugin. This is not exactly clear.

Reply:
It is about the plugin, so I've just changed it to _"classes, styles or attributes that are defined by another plugin, which **has** (...)"_. Thanks.

Reply:
Cool, thanks!